app/views/declaration/summary/summary_page.scala.html

```diff
@@ -131,6 +131,6 @@
     @if(mode == Mode.Draft) {
         <p class="mt-5">@messages("saved.declarations.summary.paragraph")</p>
 
-        <a class="button mt-3" href="@routes.DispatchLocationController.displayPage(mode)">@messages("saved.declarations.summary.continue")</a>
+        <a class="button mt-3" href="@routes.DispatchLocationController.displayPage()?mode=@mode.name">@messages("saved.declarations.summary.continue")</a>
```

Comment:
As mentioned above, not keen on having this query param everywhere. What is the route for display page already had query params?

Reply:
Sorry i miss this again - i think mode should not have default value. We can remove this from router - it is already done on level of binder.